dashboard/test/controllers/survey_results_controller_test.rb

```diff
@@ -10,13 +10,14 @@ class SurveyResultsControllerTest < ActionController::TestCase
     sign_in @teacher
 
     assert_creates(SurveyResult) do
-      post :create, {survey: {survey2016_ethnicity_asian: 22, survey2016_foodstamps: 3}, format: :json}
+      post :create, {survey: {kind: 'Diversity2016', survey2016_ethnicity_asian: 22, survey2016_foodstamps: 3}, format: :json}
```

Comment:
Seems reasonable if it's going to remain independent of the property name, but does it also get whitelisted on the server side?